storage/innobase/xtrabackup/test/t/PXB-787_tablespace_corruption.sh

```diff
@@ -0,0 +1,17 @@
+# PXB-787 - xtrabackup continues after finding corrupted tablespace.
```

Comment:
Please use the lower case name for filenames. I think bug number in the file name is not necessary (old though we had only numbers). Just tablespace_corruption.sh is good enough.